cartographer_ros/src/cartographer_node_main.cc

```diff
@@ -524,26 +524,26 @@ void Node::PublishSubmapList(const int64 timestamp) {
 void Node::PublishPose(const int64 timestamp) {
   const ::cartographer::common::Time time =
       ::cartographer::common::FromUniversal(timestamp);
-  const Rigid3d tracking_to_map = trajectory_builder_->pose_estimate().pose;
+  const Rigid3d tracking_to_local = trajectory_builder_->pose_estimate().pose;
```

Comment:
As discussed offline, we'll instead rename GetOdometryToMapTransform() to GetLocalToGlobalTransform() in future PRs.